pkgs/applications/audio/cider/default.nix

```
@@ -0,0 +1,33 @@
+{ appimageTools, lib, fetchurl }:
+
+appimageTools.wrapType2 rec {
+  pname = "cider";
+  version = "1.3.1308";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url =
+      "https://1308-429851205-gh.circle-artifacts.com/0/%7E/Cider/dist/artifacts/Cider-${version}.AppImage";
```

```suggestion
    url = "https://1308-429851205-gh.circle-artifacts.com/0/%7E/Cider/dist/artifacts/Cider-${version}.AppImage";
```
You are sure this URL stays similar?

---

I'm not sure, no. But it has thus far.